chia/util/config.py

```diff
@@ -162,3 +164,34 @@ def traverse_dict(d: Dict, key_path: str) -> Any:
         return val
     else:
         raise KeyError(f"value not found for key: {key}")
+
+
+start_methods: Dict[Optional[str], Optional[str]] = {
+    None: None,
+    "default": None,
+    "fork": "fork",
+    "forkserver": "forkserver",
+    "spawn": "spawn",
+}
+
+
+def process_config_start_method(
+    method: object,
+    log=logging.Logger,
+) -> Union[None, Literal["fork"], Literal["fork_server"], Literal["spawn"]]:
+    sentinel = object()
```

Comment:
This is confusing.

Reply:
It's an alternative to `None` since `None` is presently a valid value.